packages/Accordion/src/index.tsx

```diff
@@ -1,34 +1,47 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import AccordionContent from "./AccordionContent";
 import AccordionContentDetails from "./AccordionContent/AccordionContentDetails";
 import AccordionContentFooter from "./AccordionContent/AccordionContentFooter";
 import AccordionHeader from "./AccordionHeader";
 
 interface IAccordionProps {
   children: [JSX.Element, JSX.Element];
+  open?: boolean;
 }
 
-const Accordion = ({ children }: IAccordionProps): JSX.Element => {
-  const flex = 'flex';
-  const none = 'none';
+const Accordion = ({ children, open }: IAccordionProps): JSX.Element => {
+  const flex = "flex";
```

Comment:
Should we move the constants to a constants.ts file?

Reply:
fixed, thanks

> Is it worthy to use the EventBus as an util? Is it going to be used across components?

yes, because we are using it on textBlock, drag and drop and Accordion... you can import it to where you need it